RealmSwift/AnyRealmValue.swift

```diff
@@ -156,4 +178,61 @@ public enum AnyRealmValue: Hashable {
     public init() {
         self = .none
     }
+
+    /// Returns a `AnyRealmValue` storing a `Map`.
+    ///
+    /// - Parameter dictionary: A Swift's dictionary of `AnyRealmValue` values.
+    /// - Returns: Returns an `AnyRealmValue` storing a `Map`.
+    public static func fromDictionary(_ dictionary: Dictionary<String, AnyRealmValue>) -> AnyRealmValue {
+        let map = Map<String, AnyRealmValue>()
+        dictionary.forEach { (key, value) in
+            map.setValue(value, forKey: key)
+        }
```

Comment:
```suggestion
        map.merge(dictionary, uniquingKeysWith: { $1 })
```

`setValue(forKey:)` is the obj-c KVC function, so it involves a bunch of funky bridging compared to using the subscript operator.